compiler/rustc_middle/src/mir/statement.rs

```diff
@@ -836,7 +836,7 @@ impl BorrowKind {
 
     /// Returns whether borrows represented by this kind are allowed to be split into separate
     /// Reservation and Activation phases.
-    pub fn allows_two_phase_borrow(&self) -> bool {
+    pub fn is_two_phase_borrow(&self) -> bool {
```

Comment:
in some sense its *always* a two phase borrow, just sometimes its activated at the same point its created I think?

The previous name confused me because it made it sound like a reference *may or may not* be a two phase borrow and then we went on to unconditionally treat it as two phase without ever checking if it "actually is".